federation-2/harmonizer/RELEASE_CHECKLIST.md

```diff
@@ -23,8 +23,13 @@ None of the `federation-rs` packages currently maintain changelogs as they are l
 
 1. Once merged, run `git checkout main && git pull`
 1. Run `cargo xtask tag --package $PUBSLUG --real-publish`
+   - **NOTE** If you get an error, you might need to run `git fetch --tags -f` to ensure the remote tags match the local tags
```

Comment:
this _should_ happen automatically in xtask tag ([src](https://github.com/apollographql/federation-rs/blob/main/xtask/src/tools/git.rs#L94))- did you run into this issue? i am of the belief (perhaps a mistaken belief) that if you run `git fetch --tags -f` it will overwrite any local tags with the tags on the remote, but it will not delete any local tags that you have on your machine. So, if you had a local tag that you didn't want to push, `git fetch --tags -f` would leave you in a state where the next time you `git push --tags`, it would push any local tag you created. (in the past i've run into situations where tags end up on other dev machines and just, always inevitably get pushed even if you deleted them in the past. they are very sticky, ick... thus, this little ritual) 

so in `xtask tag` we actually do the following (there may be a better way):

1) list all the tags on the local machine
2) iterate through those and delete every single one of them until there are none left, kill them with fire etc. etc.
3) fetch the tags from the remote
4) create another tag locally
5) push the tag, ensuring it is only one tag that is being pushed at a time